x-pack/plugins/actions/server/constants/event_log.ts

```diff
@@ -9,6 +9,5 @@ export const EVENT_LOG_PROVIDER = 'actions';
 export const EVENT_LOG_ACTIONS = {
   execute: 'execute',
   executeStart: 'execute-start',
-  executeViaHttp: 'execute-via-http',
```

Comment:
Typically different event log actions means a different document is written out (we write out separate docs for `execute` and `execute-start`. In this case, it doesn't seem to make sense to write out a different doc for `execute-via-http` and it made more sense to store the source in the `execute` doc. This value was unused anyway, so I removed it.